Content.IntegrationTests/Tests/Station/EvacShuttleTest.cs

```diff
@@ -0,0 +1,111 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Shuttles.Components;
+using Content.Server.Shuttles.Systems;
+using Content.Server.Station.Components;
+using Content.Shared.CCVar;
+using Content.Shared.Shuttles.Components;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Map.Components;
+
+namespace Content.IntegrationTests.Tests.Station;
+
+[TestFixture]
+[TestOf(typeof(EmergencyShuttleSystem))]
+public sealed class EvacShuttleTest
+{
+    /// <summary>
+    /// Ensure that the emergency shuttle can be called, and that it will travel to centcomm
+    /// </summary>
+    [Test]
+    public async Task EmergencyEvacTest()
+    {
+        await using var pair = await PoolManager.GetServerClient(new PoolSettings { DummyTicker = true, Dirty = true });
+        var server = pair.Server;
+        var entMan = server.EntMan;
+        var ticker = server.System<GameTicker>();
+
+        var shuttleEnabled = pair.Server.CfgMan.GetCVar(CCVars.EmergencyShuttleEnabled);
+        pair.Server.CfgMan.SetCVar(CCVars.GameMap, "Saltern");
+        pair.Server.CfgMan.SetCVar(CCVars.GameDummyTicker, false);
+        pair.Server.CfgMan.SetCVar(CCVars.EmergencyShuttleEnabled, true);
+
+        await server.WaitPost(() => ticker.RestartRound());
+        await pair.RunTicksSync(25);
```

Comment:
is restarting round always guaranteed to be under 25 ticks i can see this making tests randomly fail because the vm was slower than usual

Reply:
How slow the PC is shouldn't be affecting tick times?

Reply:
well 25 still seems very arbitrary

Reply:
@deltanedas AFAIK 25 is very arbitrary. IIRC I've just been using it in various tests out of habit because that's the what `TestPair.ReallyBeIdle()` does, though I have no idea where the number comes from and it probably wasn't required here. E.g., `TestPair.CLeanPooledPair()` just restarts rounds with
```cs
await Server.WaitPost(() => gameTicker.RestartRound());
await RunTicksSync(1);
```
I know some round restart shitcode does (or used to?) use `Timer.Spawn()` to delay things like player entities spawning, so there probably is some minimum number of ticks that are needed. And unless theres some shitcode code somewhere that relies on wall time or something, the slowdown shouldn't matter.

The one instance were a specific number of ticks are needed is for networking. E.g., if you spawn an entity on the server, it needs (IIRC) 3 ticks before the entity appears on the client, though I usually just wait 5 ticks which also gives some padding in case the test pair networking ever changes for whatever reason.